content/communities/_index.md

```diff
@@ -1,2 +1,43 @@
 +++
+template = "map.html"
 +++
+
+# The community of research software engineers in the Nordics
+
```

Comment:
Around here, I would maybe add some sort of temporal reference here, e.g. "since becoming an association in October 2021", or "as of Spring 2022, 70 members are in our mailing list"... you see the gist. It gives the idea that the network is fresh and recent.